### DISCUSSION TITLE
howto have webhook trigger a .sh script

### DISCUSSION THREAD

**Discussion by asdffdsazqqq** [jellyfin/jellyfin#9576] [General] [answered]
hello, thanks, first time poster,

how to get a webhook to run a local. sh script
based on the docs and searching, i need create a local web server and have that server run the script.

is there a simpler method to run a local .sh script?

thanks much,
david

**Comment by crobibero** [comment, marked as answer]
That isn’t something that is currently supported. Feature requests belong at https://features.jellyfin.org
